### ISSUE TITLE
Sdl_gfx

### ISSUE THREAD

**Issue by bararchy** [ysbaddaden/sdl.cr#20]
Hi, is there a chance you will add the binding for the gfx lib? 
It allows for a Much better handling of polygons and drawing shapes by points

**Comment by ysbaddaden** [comment]
I didn't know about this plugin, so I'm not sure.

**Comment by bararchy** [comment]
As you feel, I'm right now kinda lost as to figure out if I use SDL and\or SFML.

**Comment by ysbaddaden** [comment]
Yeah, so many choices: https://www.khronos.org/opengl/wiki/Related_toolkits_and_APIs

**Comment by bararchy** [comment]
@ysbaddaden Oh no, why did you give me this link! more choices...

**Comment by ysbaddaden** [comment]
SFML is supposed to be "simple for the newcomer", if it has a nice graphics library, try it out.

Otherwise SDL with an OpenGL graphics library may prove good :)

**Comment by bararchy** [comment]
@ysbaddaden Seems I'm not the only one interested in this: https://github.com/rumenzu/sdl.cr/commit/352eb2d3776f30ef18e8efe6dc8b7717032c94f8  
Care to re-think?

**Comment by bararchy** [comment]
@rumenzu would you care to open a PR for the GFX binding?

**Comment by rumenzu** [comment]
@bararchy I was planning to actually. 

Only graphic primitives are available though. And I defined only a few methods for `Renderer`.
I'm currently in a workshop but will definitely send a PR asap.

**Comment by bararchy** [comment]
@rumenzu Awesome! thanks :)
